src/tasks/includes/utilities.js

```diff
@@ -13,14 +15,35 @@ const Promise = require('bluebird');
 const utilities = {
 
   /**
-   * Generic error handler for streams called in `watch` tasks (used by gulp-plumber)
+   * Handles the output for any errors that might have been captured
+   * during the build and zip Gulp tasks.
+   *
+   * @memberof slate-cli.utilities
+   */
+  outputErrors: () => {
+    if (!errors.length) {
```

Comment:
I'm cool with that, but aren't we doing this for other instances - e.g. https://github.com/Shopify/slate-tools/blob/master/src/tasks/includes/utilities.js#L166